scipy/signal/tests/test_waveforms.py

```diff
@@ -290,62 +291,62 @@ def test_sweep_poly_cubic3(self):
 class TestGaussPulse:
 
     def test_integer_fc(self):
-        float_result = waveforms.gausspulse('cutoff', fc=1000.0)
-        int_result = waveforms.gausspulse('cutoff', fc=1000)
+        float_result = _multimethods.gausspulse('cutoff', fc=1000.0)
+        int_result = _multimethods.gausspulse('cutoff', fc=1000)
         err_msg = "Integer input 'fc=1000' gives wrong result"
         assert_equal(int_result, float_result, err_msg=err_msg)
 
     def test_integer_bw(self):
-        float_result = waveforms.gausspulse('cutoff', bw=1.0)
-        int_result = waveforms.gausspulse('cutoff', bw=1)
+        float_result = _multimethods.gausspulse('cutoff', bw=1.0)
+        int_result = _multimethods.gausspulse('cutoff', bw=1)
         err_msg = "Integer input 'bw=1' gives wrong result"
         assert_equal(int_result, float_result, err_msg=err_msg)
 
     def test_integer_bwr(self):
-        float_result = waveforms.gausspulse('cutoff', bwr=-6.0)
-        int_result = waveforms.gausspulse('cutoff', bwr=-6)
+        float_result = _multimethods.gausspulse('cutoff', bwr=-6.0)
```

Comment:
Can't we do just `gausspulse`? I mean is it possible?

Reply:
I think yes, we should be able to do that.